api/v1/verticadb_webhook.go

```diff
@@ -1902,18 +1906,19 @@ func (v *VerticaDB) validateTerminatingSandboxes(old runtime.Object, allErrs fie
 }
 
 // checkSboxForShutdown checks if sbox or its scluster has shutdown field set to true in spec/status
-func (v *VerticaDB) checkSboxForShutdown(newSandbox *Sandbox, newSClusterMap map[string]*Subcluster, statusSClusterIndexMap map[string]int) string {
+func (v *VerticaDB) checkSboxForShutdown(newSandbox *Sandbox, newSClusterMap map[string]*Subcluster, statusSClusterIndexMap map[string]int) []string {
+	errMsgs := []string{}
 	if newSandbox.Shutdown {
-		return fmt.Sprintf("Shutdown field of sandbox %q is set to true", newSandbox.Name)
+		return append(errMsgs, fmt.Sprintf("Shutdown field of sandbox %q is set to true", newSandbox.Name))
 	}
 
 	for _, subcluster := range newSandbox.Subclusters {
 		i, foundInStatus := statusSClusterIndexMap[subcluster.Name]
 		if foundInStatus && (newSClusterMap[subcluster.Name].Shutdown || v.Status.Subclusters[i].Shutdown) {
-			return fmt.Sprintf("Subcluster %q is marked for shut down or has already been shut down", subcluster.Name)
+			return append(errMsgs, fmt.Sprintf("Subcluster %q is marked for shut down or has already been shut down", subcluster.Name))
```

Comment:
Do not return.
Also to make the error msg shorter, we can first append the subcluster names and then add the msg. The err msg will look like: ......Subclusters sc1, sc2, sc3 are marked for shut down or have already been shut down.